app/src/main/cpp/skyline/os.h

```diff
@@ -25,13 +25,14 @@ namespace skyline::kernel {
         std::shared_ptr<type::KProcess> process; //!< The KProcess object for the emulator, representing the guest process
         service::ServiceManager serviceManager; //!< This manages all of the service functions
         MemoryManager memory; //!< The MemoryManager object for this process
+        std::string appFilesPath; //!< The full path to the apps data directory
```

Comment:
`dataDirectory` is better imo

Reply:
It isn't the app data dir though

Reply:
Wait, what else was "apps data directory" supposed to refer to ?

Reply:
It's the app's files directory... Where did you read that?

Reply:
You should re-read the line in question 😛

Reply:
Oh....
Blame the GitHub app